create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+JEKYLL_ASSETS_DIR=$HOME'/Documents/portfolio/assets/'
+JEKYLL_POSTS_DIR=$HOME'/Documents/portfolio/_posts/'
+
+FOLDER_NAME='';
+TITLE='';
+DETAIL='';
+DESCRIPTION='';
+VID=false;
+IMG=false;
+
+while [ -n "$1" ]; do 
+
+    case "$1" in 
+    -from)
+        dir="$2"
+        echo "Copying files from ${dir} to ${JEKYLL_ASSETS_DIR}"
+        cd ${JEKYLL_POSTS_DIR}
+        FOLDER_NAME=${dir##*/}
+        mkdir ${FOLDER_NAME}
+        cp -rf ${dir} ${JEKYLL_ASSETS_DIR}
+        
+        shift
+        
+        ;;
+    -title)
+        TITLE="$2"
+        shift
+        
+        ;;
+    -detail)
+        DETAIL="$2"
+        shift
+
+        ;;
+    -description)
+       DESCRIPTION="$2"
+       shift
+
+        ;;
+    -img)
+        IMG=true;
+
+        ;;
+
+    -vid)
+       VID=true;
+
+       ;;
+    *) echo "Not valid option"
+      break
+        ;;
+    esac
+
+    shift
+done
+
+TITLE_STRIPPED=${TITLE// /_}
+DATE=`date +%Y-%m-%d`
```

Comment:
I tend to use `$(...)` to run subcommands. This isn't a big deal, but it nests nicely, and is more visually apparent.